src/EditorFeatures/Core/Implementation/EditAndContinue/ActiveStatementTrackingService.cs

```diff
@@ -165,7 +165,7 @@ internal Task TrackActiveSpansAsync()
             {
                 try
                 {
-                    return Task.Run(() => TrackActiveSpansAsync(_cancellationSource.Token), _cancellationSource.Token);
+                    return Task.Run(() => TrackActiveSpansAsync(_cancellationSource.Token), _cancellationSource.Token).ReportNonFatalErrorAsync();
```

Comment:
😕 Why is the caller not responsible for this, considering the task in question is returned?

Reply:
You are right. Should be the caller.

Reply:
❕ Ah, yes now I am sure. This should be on the caller not here. The caller _looks_ like it isn't handling errors correctly.

Reply:
yes

Reply:
Fixed.

Reply:
Hit the merge button since it was green not realizing you haven't signed off yet. If you have more feedback, I'll be happy to follow up.